src/main/java/com/google/neighborgood/servlets/UserInfoServlet.java

```diff
@@ -38,14 +39,53 @@
 public class UserInfoServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     UserService userService = UserServiceFactory.getUserService();
 
+    // Retrieves user accounts for the topscorers board
+    if (request.getParameterMap().containsKey("action")
+        && request.getParameter("action").equals("topscorers")) {
```

Comment:
Any thoughts on breaking out the contents of this if {...} to a separate function just for keeping this function small and easier for readers to understand what is going on in it?

Reply:
That's a good idea, thanks! I will create a private function that executes the code inside the If-block.